### PR TITLE
Update AOT limitations /8

### LINKED ISSUE #30908: ASP.NET Core and native AOT compatibility
### Description

Add content from [ASP.NET Core and native AOT compatibility](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-preview-3/#asp-net-core-and-native-aot-compatibility#asp-net-core-and-native-aot-compatibility)

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/native-aot?view=aspnetcore-8.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/native-aot.md

### Document ID

8839fa1e-7c10-4b22-d03d-669906a2b9ea

### Article author

mitchdenny

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30908

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/native-aot.md](https://github.com/dotnet/AspNetCore.Docs/blob/db6840f83b62a0a2bfccdb40de6a74388ea73c57/aspnetcore/fundamentals/native-aot.md) | [ASP.NET Core support for native AOT](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/native-aot?branch=pr-en-us-30966) |

### CODE REVIEW

**Review by tdykstra — APPROVED** [review]
LGTM. One nit suggestion.

**Comment by Rick-Anderson** [comment]
> LGTM. One nit suggestion.

I don't see the nit suggestion.

File: aspnetcore/fundamentals/native-aot.md [hunk 1]
```diff
@@ -62,9 +58,15 @@ Not all features in ASP.NET Core are currently compatible with native AOT. The f
...
 | WebSockets | <span aria-hidden="true">✔️</span><span class="visually-hidden">Fully supported</span> | | |
 
-It's important to test an app thoroughly when moving to a native AOT deployment model. The AOT deployed app should be tested to verify functionality hasn't changed from the untrimmed and JIT-compiled app. When building the app, review and correct AOT warnings. An app that issues AOT warnings during publishing is not guaranteed to work correctly. If no AOT warnings are issued at publish time, the published AOT app should work the same as when it's run in development.
+For more information on limitations, see:
+
+* [Limitations of Native AOT deployment](/dotnet/core/deploying/native-aot#limitations-of-native-aot-deployment)
+* [Introduction to AOT warnings](/dotnet/core/deploying/native-aot/fixing-warnings)
+* [Known trimming incompatibilities](/dotnet/core/deploying/trimming/incompatibilities)
+* [Introduction to trim warnings](/dotnet/core/deploying/trimming/fixing-warnings)
+* [This GitHub issue](https://github.com/dotnet/core/issues/8288)
```
Comment (tdykstra) [inline aspnetcore/fundamentals/native-aot.md:67]:
```suggestion
* [GitHub issue dotnet/core #8288](https://github.com/dotnet/core/issues/8288)
```

**Review by tdykstra — APPROVED** [review]
I don't know what happened to it the first time, but here it is again.

**Pull request merged**